src/frontend/src/tests/test-support/test-data/projects.stub.ts

```diff
@@ -60,6 +64,7 @@ export const exampleProject1: Project = {
     }
   ],
   duration: 8,
+  endDate: new Date('07/21/21'),
```

Comment:
all of these end dates need to be the latest end date of the project's work packages